backend/src/main/kotlin/com/clouway/app/LogsObserver.kt

```diff
@@ -0,0 +1,20 @@
+package com.clouway.app
+
+import com.clouway.app.core.Observer
+import org.apache.log4j.Logger
+
+/**
+ * @author Stanimir Iliev <stanimir.iliev@clouway.com>
+ */
+
+class LogsObserver(private val logger: Logger) : Observer {
```

Comment:
Observer sounds too common - observer of what ? the names of the objects and interfaces should be describe what are they responsible  and what are they doing  

in this case we listen for an events over User : registration event, login event  ...  so we can named the interface UserChangeListener UserEvnetsListener .. . UserEvnetsHandler is suitable too.

The name of the class can be UserChangeLogger - the name descirbes that this is a logger object of a user events. 
Reading the name we can understand the class purpose.